extensions/unducklink/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "unducklink",
+  "title": "Unduck.link",
+  "description": "Access Theo's (t3.gg) unduck.link search engine through Raycast",
```

Comment:
This is certainly not a search engine. Would be great to have something less misleading in the description.